### PR TITLE
Updating the UI

### PR DESCRIPTION
**Pull request by vzvenyach**

### CODE REVIEW

**Comment by vzvenyach** [comment]
Current view of the front page:
 
screen shot 2015-11-28 at 9 51 08 am

**Comment by vzvenyach** [comment]
We have a working bid detail page, including a flash message when you have a successful bid. Screenshot below.

screen shot 2015-11-28 at 1 45 42 pm

**Comment by vzvenyach** [comment]
Ok. I think this is ready for review.

**Comment by allalala** [comment]
@ddotrollins can you also take a quick look at these QX adjustments and see if you spot any glaring issues?

**Comment by ddotrollins** [comment]
@allalala looks good, I would remove the decimal places on the bid amount to read "$2,000" vs. "2,000.00" to reinforce only bidding in dollar increments. Otherwise is fine from a functional standpoint, we can make CSS updates later.

**Comment by vzvenyach** [comment]
@adelevie is that something enforced at the model level? If not, I can probably format it at the template level.

**Comment by adelevie** [comment]
The db is capable of decimals/floats, but the app prevents it. The relevant
files would be app/models/presenter/bid.rb and app/models/place_bid.rb.

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: bundle exec rails server -p $PORT
+web: bundle exec rails s
```
Comment (adelevie) [inline Procfile]: @vzvenyach why this change? AFAICT, Cloud Foundry needs that $PORT env var.

**Comment by adelevie** [comment]
rebased
